### DISCUSSION TITLE
Connection Failed: br-connection-profile-unavailable

### DISCUSSION THREAD

**Discussion by panicd7** [blueman-project/blueman#2391] [Q&A] [answered]
blueman/stable,now 2.3.5-2+b1 amd64 [installed]
bluetooth/stable,stable-security,now 5.66-1+deb12u1 all [installed]
bluez/stable,stable-security,now 5.66-1+deb12u1 amd64 [installed]
bluez-firmware/stable,now 1.2-9 all [installed]
bluez-obexd/stable,stable-security,now 5.66-1+deb12u1 amd64 [installed]
bluez-tools/stable,now 2.0~20170911.0.7cb788c-4 amd64 [installed]
libbluetooth3/stable,stable-security,now 5.66-1+deb12u1 amd64 [installed,automatic]
libspa-0.2-bluetooth/stable,now 0.3.65-3+deb12u1 amd64 [installed]
libspa-0.2-modules/stable,now 0.3.65-3+deb12u1 amd64 [installed,automatic]
pulseaudio/stable,now 16.1+dfsg1-2+b1 amd64 [installed]
pulseaudio-utils/stable,now 16.1+dfsg1-2+b1 amd64 [installed,automatic]
xfce4-pulseaudio-plugin/stable,now 0.4.5-1 amd64 [installed,automatic]

Distribution: Debian deb 6.1.0-17-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.69-1 (2023-12-30) x86_64 GNU/Linux
Desktop environment: xfce4/stable,now 4.18 all 

Problem exist only when trying to connect a new bluetooth portable speaker : 
alias Tronsmart T7 Mini
class 0x240404
appearance 0x0000
paired yes, trusted yes, blocked no, Legacy Pairing no, Connected no
UUIDs
0000110a-0000-1000-8000-00805f9b34fb Audio Source
0000110b-0000-1000-8000-00805f9b34fb Audio Sink
0000110c-0000-1000-8000-00805f9b34fb Remote Control Target
0000110e-0000-1000-8000-00805f9b34fb Remote Control
0000111e-0000-1000-8000-00805f9b34fb Handsfree
00001200-0000-1000-8000-00805f9b34fb PnP Information
00006666-0000-1000-8000-00805f9b34fb Unknown
 adapter /org/bluez/hci0

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

Error when $ bluetoothctl connect F5:4E:FD:EE:0E:CC
Attempting to connect to F5:4E:FD:EE:0E:CC
Failed to connect: org.bluez.Error.Failed br-connection-profile-unavailable

**Comment by panicd7** [comment, marked as answer]
Problem solved for pulseaudio removed pipewire
#apt purge libspa-0.2-modules
#apt autoremove
#apt install pulseaudio-module-bluetooth libspa-0.2-bluetooth
$killall pulseaudio
#service bluetooth restart

**Reply by Mike-Bravo** [reply]
it worked for me. thanks

**Reply by pushpak25w** [reply]
thanks alot @panicd7

**Comment by samusz** [comment]
yes ! Installing `libspa-0.2-bluetooth` is what was missing for me (Debian, KDE box)
